learn/guides/graphql-client-tool.md

```diff
@@ -14,69 +14,154 @@ redirect_from:
   - /learn/graphql-client-tool
 --- 
 
-GraphQL schemas of a service are now most often specified using what’s known as the GraphQL SDL (schema definition language). It is also sometimes referred to as just GraphQL schema language. Ballerina GraphQL client tool will make it easy for you to generate a client in Ballerina given the GraphQL schema (SDL) and GraphQL queries. You can generate multiple clients in Ballerina for multiple GraphQL documents for a given GraphQL SDL. It also enables you to generate multiple Ballerina modules for multiple GraphQL projects to work with different GraphQL APIs. 
+The GraphQL schema is necessary to validate the GraphQL queries. Each GraphQL query will be validated against the given GraphQL schema before generating a Ballerina client remote operation and the corresponding Ballerina record types. You can use the generated Ballerina client to execute queries/mutations on the relevant GraphQL API through the generated remote operations. 
 
-The Ballerina GraphQL client tool support provides the following capabilities.
+For advanced use cases, you can generate multiple clients using multiple GraphQL documents or generate multiple Ballerina modules per GraphQL API endpoint.
+
+## Generate a client from a GraphQL config
 
 > **Prerequisites:** Install the [GraphQL Foundation VSCode plugin](https://marketplace.visualstudio.com/items?itemName=GraphQL.vscode-graphql) and the latest [Ballerina Swan Lake distribution](https://ballerina.io/downloads/).
 
-1. Generate a Ballerina client from a given GraphQL config file configured with a GraphQL schema (SDL) and a GraphQL document using the CLI tool.
+The steps below demonstrates how to generate a Ballerina client from a GraphQL config file using the `graphql` CLI tool.
 
-2. Generate multiple Ballerina clients from a given GraphQL config file configured with a GraphQL SDL and multiple GraphQL documents using the CLI tool. Each document will generate a separate Ballerina client.
+### Create a Ballerina package
 
-3. Generate multiple Ballerina modules from a given GraphQL config file configured with multiple GraphQL projects. Each project will generate a separate Ballerina module. This enables you to work with multiple GraphQL APIs by configuring each GraphQL API under a separate project.
+Create a Ballerina package using the `bal new` command.
```

Comment:
```suggestion
Follow the instructions below to create a Ballerina package using the `bal new` command.
```
We can remove the sub-sections and have numbered steps if that is more user-friendly.

Reply:
The following instructions won't create the Ballerina package. They customize the package generated using the `bal new` command. Therefore, the above change will not be done.